app/com/arpnetworking/metrics/portal/scheduling/impl/JobScheduler.java

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright 2018 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.metrics.portal.scheduling.impl;
```

Comment:
Probably doesn't need to be in an `impl` sub-package; or at least I'd like to understand why this is and `JobRef` is not.

Reply:
The reason is that I am not clear on the rules for whether something belongs in an "impl" subpackage. :P